libavfilter/dnn/dnn_backend_tf.c

```diff
@@ -154,7 +154,7 @@ static DNNReturnType get_input_tf(void *model, DNNData *input, const char *input
     TF_DeleteStatus(status);
 
```

Comment:
this patch is more likely to be multiple output support for tf backend, not just 4, not just detection

Reply:
changed it to two patches